### DISCUSSION TITLE
分享你打包的好玩的 APP

### DISCUSSION THREAD

**Discussion by tw93** [tw93/Pake#74] [Ideas]
大家可以在这个讨论下面交流分享自己打包的好玩的 APP~
可以将自己的配置贴出，如 tauri.conf.json 改动点、pake.js 改动点，这样方便其他同学也去玩玩

**Comment by QuentinHsu** [comment]
# CodePen

## Change Log

### 2022-11-08 v0.0.1

- 仅打包 CodePen 网页，未做任何修改

### 2022-11-09 v0.0.2

- 隐藏 Pen 界面的 CodePen Pro AD （CSS 实现）
- 规避 Pen 界面中左上角内容被窗口按钮挡住 （CSS 实现）

⏬ [Release](https://github.com/QuentinHsu/Pake/releases)

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/21162238/200725108-e4ac7f69-ef22-48ec-ae52-6645ac10b9e7.png">

**Reply by tw93** [reply]
之前也有朋友在 issue 中有一些，可见 https://github.com/tw93/Pake/issues/59

**Comment by huangjiahua** [comment]
# github.dev 

没有任何修改。本来准备试试 vscode.dev。但是好像 popup 支持不太好没办法登录 github，而且 webview 不能像 chrome 操作本地文件，就没什么用了。

<img width="1880" alt="Screenshot 2022-11-11 at 8 02 34 PM" src="https://user-images.githubusercontent.com/26034312/201456203-a6f3299f-111d-4c65-9681-6159619d0046.png">


[Release](https://github.com/huangjiahua/Pake/releases/download/bundle/GitHub-Dev.dmg)

**Comment by cylllllll** [comment]
仅打包 gcores 网页
![IMAGE 2022-11-14 1:38:39 PM](https://user-images.githubusercontent.com/17983727/201583671-210c8c23-87a6-4e6a-ba8f-60dca834ab69.jpg)
![IMAGE 2022-11-14 1:38:44 PM](https://user-images.githubusercontent.com/17983727/201583683-645d92c1-750b-4187-bb57-e494af98cf6b.jpg)

**Comment by DrayChou** [comment]
打包 Windows 版的 WeRead 和 MoeCat(长毛象分站)
https://github.com/draychou/Pake/releases

![image](https://user-images.githubusercontent.com/760245/202700996-a1b9030d-dcad-4e8d-8e9a-f07d0fb2efcc.png)

**Comment by turkyden** [comment]
求打包李志 App https://lizhi.turkyden.com/

**Reply by tw93** [reply]
早有想法 我也很喜欢李志 到时候有一些页面优化的怎么联系你 有电报吗

这个页面的live视频我看了看不太能播放，此外下载功能有些问题，看看如何优化一趴，此外有一些外跳的地址我标注出来了，想着体验好一些要不要对于pake的场景先去掉，比如说我在url后面带一个参数，来辨别一下，如 from = pake  https://lizhi.turkyden.com/?from=pake

<img width="1312" alt="image" src="https://user-images.githubusercontent.com/8736212/206667919-1a38f422-0089-4371-a02c-02ef3c2e3d3b.png">

**Reply by tw93** [reply]
已经打包，放到首页了，大伙直接下载即可

**Comment by github-logic** [comment]
FlowUs笔记
只有deb运行正常，Appimage打开不了
[FlowUs.deb](https://www.123pan.com/s/vK5A-W2RP3)
![AjmtZ.png](https://i.328888.xyz/2022/12/20/AjmtZ.png)

**Reply by Aurora-Dylan** [reply]
请问有msi的吗？

**Comment by hdward9** [comment]
能帮忙打包一下icloud网页吗，经常在其他平台上用到备忘录

**Reply by tw93** [reply]
空闲时安排一趴 假如你着急可以试试这个  https://github.com/tw93/Pake/wiki/GitHub-Actions-%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%E5%A4%9A%E7%B3%BB%E7%BB%9F%E7%89%88%E6%9C%AC

不会技术也可以自己弄

**Reply by hdward9** [reply]
好的👌

**Comment by tw93** [comment]
[ZLibrary.dmg.zip](https://github.com/tw93/Pake/files/10328823/ZLibrary.dmg.zip)

**Reply by wuliring** [reply]
有windows版本的吗？

**Comment by ailove111111** [comment]
学习

**Reply by weqoocu** [reply]
https://wx.qq.com/

**Comment by einverne** [comment]
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/1962738/210916327-de7caf33-c363-4ae5-a54f-3efd7ec6ee3f.png">

个人非常喜欢的一个翻译工具 [Papago](https://papago.naver.com/)，个人体验起来比 Google Translate，和 DeepL 都顺手一些，打包了一下。

可以到这里下载 https://github.com/einverne/Pake/releases/tag/V0.1.1

**Reply by AoryZen** [reply]
大佬，能否打包一下deepl和谷歌翻译呀，万分感谢

**Comment by ailove111111** [comment]
谢谢🙏

Ein Verne ***@***.***>于2023年1月6日 周五10:18写道：

**Comment by lmm214** [comment]
### Pake x 微信读书 x 霞鹜文楷

![pake-wr-3](https://user-images.githubusercontent.com/1472390/211195603-4ea1efd8-0e21-436b-b664-109bafa49fad.png)

![pake-wr-4](https://user-images.githubusercontent.com/1472390/211195609-0d2d95c4-592f-427f-a2d2-324d7a44c003.png)

- 加上了 「霞鹜文楷」在线字体；
- 增加内容宽度；
- 覆盖了亮色主题背景色为淡米色；
- 隐藏右侧控制按钮，只保留字体大小按钮；
- 隐藏底部工具栏，鼠标移动至底部显示。

###  下载地址

<https://github.com/lmm214/Pake/releases>

### 改动代码

```
window.addEventListener("DOMContentLoaded", (_event) => {
  //在 head 用添加 link 在线字体
  const link = document.createElement("link");
  link.rel = "stylesheet";
  link.type = "text/css";
  link.href = "https://npm.elemecdn.com/lxgw-wenkai-screen-webfont/style.css";
  document.head.append(link);


  const style = document.createElement("style");
  style.innerHTML = `
// CSS 修改字体
    *{font-family: "LXGW WenKai Screen" !important;}    
//去除 body 滚动条
    body::-webkit-scrollbar{width:0px;height:0px;}
//修改文本两侧边距
    .readerChapterContent {
        margin-left: 2rem;
        margin-right: 2rem;
    }
//字体控制默认透明显示
    .readerControls_fontSize{opacity:0.3 !important;}
    .readerControls_fontSize:hover{opacity:1 !important;}
//修改亮色主题背景颜色及阴影
    .wr_whiteTheme .app_content{box-shadow: 0 8px 32px rgba(0,25,104,.1) !important ;}
    .wr_whiteTheme .app_content.shelf_container{box-shadow: none !important;}
    html body.wr_whiteTheme,
    .wr_whiteTheme .navBar,
    .wr_whiteTheme .navBar_home,
    .wr_whiteTheme .bookshelf_preview_item,
    .wr_whiteTheme .app_content,
    .wr_whiteTheme .readerControls_item,
    .wr_whiteTheme .readerControls_fontSize,
    .wr_whiteTheme .readerTopBar,
    .wr_whiteTheme .readerBottomBar,
    .wr_whiteTheme .readerBottomSettingPanel,
    .wr_whiteTheme .recommend_preview_item{
      background-color: #f9f3e8 !important;
    }
//隐藏控制条，底部 hover 显示
    .readerMemberCardTips,
    .readerTopBar,
    .readerControls_item,
    .navBarOffset {
        display: none !important;
    }
    .readerBottomBar {
        transform: none;
        display: flex !important;
        opacity: 0 !important;
        transition: opacity 400ms ease-out !important;
    }
    .readerBottomBar:hover {
        display: flex !important;
        opacity: 1 !important;
    }
    .readerBottomSettingPanel {
        display: block;
        top: 88% !important;
        height: 7% !important;
    }
```

**Reply by skyerGalaxy** [reply]
你好，我在使用过程中遇到如下问题：
- 隐藏右侧控制按钮，只保留字体大小按钮；
- 隐藏底部工具栏，鼠标移动至底部显示。
这些功能在我的电脑上不能实现
![image](https://github.com/tw93/Pake/assets/90626136/7850e528-4c7d-4514-bf71-7e103e0e67f0)
另外，上边标题跟正文内容重叠太乱了
![image](https://github.com/tw93/Pake/assets/90626136/6c9dbcff-b1bb-4b73-9ac2-2da09ed09ef6)

**Comment by cnmeeia** [comment]
![截屏2023-01-09 上午11 37 38](https://user-images.githubusercontent.com/102783643/211237359-3c385f8c-e1f3-4bb2-8766-906624ba042a.png)

**Comment by iamkissg** [comment]
得到网页版 [DeDao.dmg.zip](https://github.com/tw93/Pake/files/10785467/DeDao.dmg.zip)

拦截了新标签页打开, 可以正常点击跳转.

**Comment by wolf3c** [comment]
[poecom.app.zip](https://github.com/tw93/Pake/files/10948248/poecom.app.zip)

Quora 的 Chat AI ：poe.com。自带ChatGPT、Claude

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/9410030/224479091-b602d957-ed50-4449-b26b-e98d68b9b409.png">

**Comment by X4fun** [comment]
readwise
**[link](https://github.com/X4fun/Pake/releases/tag/V0.0.4)**
只是把readwise 网页版 打了下包... 期待更优秀的作品
![image](https://user-images.githubusercontent.com/128443521/227981741-e0a06501-88e4-4c5b-b1fe-6c8f3d48efd5.png)

**Comment by likemuuxi** [comment]
打包自用的项目（Win端）：iCloud、Readwise Reader、Github、MarkdownNice


[link](https://github.com/likemuuxi/Pake/releases/)

**Comment by 1xdali** [comment]
麻烦问一下，有人打包过Youtube Shorts嘛？

**Comment by jaysoncho92** [comment]
https://zbook.lol/     
<img width="1236" alt="image" src="https://github.com/tw93/Pake/assets/17448433/f3788e4e-b6b7-43b0-a2fd-9725d8a81cb2">

**Reply by jaysoncho92** [reply]
z-library国内镜像网站，不需要科学上网，但是目前有个问题下载的文件找不到。

**Reply by tw93** [reply]
我调试下看看

**Comment by lvwzhen** [comment]
[MagickPen.zip](https://github.com/tw93/Pake/files/11542301/MagickPen_macos.dmg.zip)

我做的一个AI写作助手：[MagickPen](https://magickpen.com/)，用Pake打包了一个客户端，太方便了😄

![SCR-20230523-pqao](https://github.com/tw93/Pake/assets/2337506/81125612-f74d-4720-88d1-d6c659c21d6f)

**Comment by wangliangliang2** [comment]
Instagram  打包 加 --transparent 会导致不能拖动窗口，该版本为未加--transparent
Discord 正常运行
两个打包都未做修改。
[Instagram.dmg.zip](https://github.com/tw93/Pake/files/11788733/Instagram.dmg.zip)
[Discord.dmg.zip](https://github.com/tw93/Pake/files/11788735/Discord.dmg.zip)

**Comment by aoannan** [comment]
https://www.meiye.art/canvas     您好 您有空可以把这个网页打包成一个APP吗？ Windows系统 感谢

**Reply by for-the-zero** [reply]
[Meiye.msi.删后缀.zip](https://github.com/Pake/files/11998136/Meiye.msi.zip)
把后缀改成msi

**Reply by aoannan** [reply]
感谢大佬

**Reply by aoannan** [reply]
为什么下载不了呀

**Comment by tiancool** [comment]
作者提供的youtube 无法全屏播放，是否可以更新一下，谢谢

**Reply by Tlntin** [reply]
Mac调了调不了，没得办法。
windows/mac通过换user-agent倒是可以全屏。

**Reply by tiancool** [reply]
> Mac调了调不了，没得办法。
> 
> windows/mac通过换user-agent倒是可以全屏。

难怪我就是mac版，可惜呀，没法全屏看视频，其他都挺好

**Comment by for-the-zero** [comment]
[Desktop.7z.001.zip](https://github.com/tw93/Pake/files/11997990/Desktop.7z.001.zip)
[Desktop.7z.002.zip](https://github.com/tw93/Pake/files/11997993/Desktop.7z.002.zip)

文件太大，分卷压缩了，删掉.zip用7zip提取

里面都是我冲浪找到的好用的pwa，包括
- 免费听歌
- 文件转换
- todo
- 文件传输
- 白板
- ps
- 免费chatgpt、newbing镜像

**Reply by AoryZen** [reply]
提取不了

**Comment by tw93** [comment]
### 微信聊天网页版
[WeChat_1.0.0_universal.dmg.zip](https://github.com/tw93/Pake/files/12104840/WeChat_1.0.0_universal.dmg.zip)

**Reply by diandianti** [reply]
请问可以打包一个linux版本的吗

我自己打包的时候提示我登录失败，我怀疑是因为requeset的header没有写对的原因，所以还想请教一下如何修改requeset的header

**Comment by 0x676e67** [comment]
ChatGPT API(bypass cloudflare) 逆向工程，自建ChatGPT网页应用打包: https://github.com/gngpp/opengpt
<img width="1365" alt="image" src="https://github.com/tw93/Pake/assets/51810656/634222a3-f281-4249-87f5-6aac785c7187">
<img width="1365" alt="image" src="https://github.com/tw93/Pake/assets/51810656/bb6415c2-80e3-41e7-bda6-834716747bbe">

**Comment by Guyungy** [comment]
AIGC知识库
---
### 介绍
中文圈最大的免费一站式人工智能知识库，学习AI知识和技巧的平台。

源网址：https://www.yuque.com/gptcn/gpt
仅打包了Win版本，方便快速查找所需的资料，使用GTP和MJ
### 下载地址
[AIGC-win.msi](https://github.com/Guyungy/AIGC-knowbase/releases/download/2023-07-28/AIGC-win.msi)

### 预览
<img src="https://github.com/Guyungy/AIGC-knowbase/blob/main/Snipaste_2023-07-28_17-03-18.png?raw=true" div align=center />

**Comment by likemuuxi** [comment]
Claude镜像打包
[Link](https://github.com/likemuuxi/Pake/releases/download/win/Claude.msi)
![image](https://github.com/tw93/Pake/assets/85612715/9c333e65-e246-41dd-86de-354f057100ab)

**Reply by loseblue** [reply]
Claude镜像打包
无法登录,无论是普通邮箱登录还是google登录。
同时间chrome的登录都是可以的。
如果有时间的麻烦修正下。谢谢。

**Reply by likemuuxi** [reply]
应该是镜像站不能使用了

**Comment by AoryZen** [comment]
能否打包一下Deepl 和 谷歌翻译，能否加入去网页广告的功能

**Comment by Guyungy** [comment]
deepl本身不就是电脑版吗

diandianti ***@***.***>于2023年7月28日 周五17:27写道：

**Reply by AoryZen** [reply]
有字数限制，网页版没遇到过

**Comment by loseblue** [comment]
claude官方网站打包
没做任何改动。
登录时需要挂美国英国梯子。google登录有点迷，不用弹出窗登录。如果不行再用弹出窗登录。
登录之后可以随意换梯子。


[Claude.zip](https://github.com/tw93/Pake/files/12417453/Claude.zip)

**Comment by xinmans** [comment]
icloud notes,苹果备忘录解决了多端同步的焦虑，windows上也可以用备忘录app了

**Comment by ryusaksun** [comment]
Perplexity
<img src="https://cdn.jsdelivr.net/gh/SUNSIR007/picx-images-hosting@master/20240326/截屏2024-03-26-18.04.21.y45p96uhb5c.png" alt="截屏2024-03-26-18" />

**Reply by ryusaksun** [reply]
https://github.com/SUNSIR007/Pake/releases/tag/1.0.0

**Comment by DongshengHuang** [comment]
求debian系的 多邻国 和 Apple Music（deb包）

**Comment by Mingtonggg** [comment]
请问windows的应用  如何使用Chrome浏览器打开呢。因为电脑上自带的edge有问题，没法挂代理。

**Comment by fangfangqiqi** [comment]
求帮忙打包一个在线动画网站 https://www.hkanime.com.hk/

**Reply by hkanime** [reply]
https://play.google.com/store/apps/details?id=com.app.hkanime

**Comment by changtekai** [comment]
[weibo.zip](https://github.com/user-attachments/files/16116926/weibo.zip)
微博 如果出现损坏提示 查看这个解决办法 https://zhuanlan.zhihu.com/p/135948430

**Comment by anghunk** [comment]
lobechat，只打包了 windows

[LobeChat.zip](https://github.com/user-attachments/files/18717491/LobeChat.zip)

**Comment by memekoudai123** [comment]
请问如何让打包的网页带上翻译？网页现在用的是沉浸式翻译，用的是LD论坛的api。

**Comment by yxnchen** [comment]
Qwen Chat 通义千问国际版 (https://chat.qwenlm.ai/)

打包了Windows版
[QwenLM.zip](https://github.com/user-attachments/files/19009352/QwenLM.zip)

![image](https://github.com/user-attachments/assets/848d9abd-6d2d-45af-94a4-3a9b446de68e)
![image](https://github.com/user-attachments/assets/1129f35e-55e7-4224-8db6-c4ef3730643b)

**Comment by yxnchen** [comment]
Overleaf  (https://www.overleaf.com/project)

打包了Windows版
[Overleaf.zip](https://github.com/user-attachments/files/19019814/Overleaf.zip)
![image](https://github.com/user-attachments/assets/f737afa6-b188-4a08-82d6-c2b9fcfa47f5)
![image](https://github.com/user-attachments/assets/0edabf59-0f1f-4ec1-a286-f1cbb44b0e31)

**Comment by Jasondeepny** [comment]
https://aistudio.google.com/  这个 mac arm 版我打包失败，大家帮忙试试看谢谢🙏

**Comment by AprDeci** [comment]
微信读书摸鱼软件 添加了全局快捷键 窗口透明 等等

[WxReader-fish](https://github.com/AprDeci/WxReader-fish)

<img width="1659" height="816" alt="all" src="https://github.com/user-attachments/assets/dfe5f1c5-8c67-486b-8efa-56cbd7b85b02" />
